pkg/operators/formatters/formatters.go

```diff
@@ -23,6 +23,8 @@ import (
 
 	"golang.org/x/sys/unix"
 
+	"github.com/dustin/go-humanize"
```

Comment:
Put these together. (I don't know why make lint isn't fixing them, it used too after 613df49979590a6e)